creator-node/src/services/stateMachineManager/processJob.js

```diff
@@ -20,7 +20,7 @@ module.exports = async function (jobName, job, jobProcessor, parentLogger) {
     result = await jobProcessor({ logger: jobLogger, ...jobData })
   } catch (error) {
     jobLogger.error(`Error processing job: ${error}`)
-    result = { error }
+    result = { error: error.message || `${error}` }
```

Comment:
i think we can just use error.message on both lines 22 and 23, no? is there ever a scenario where that doesn't exist

Reply:
if the promise gets rejected with a string instead of an Error then there won't be an error.message. example:

```js
async function test() {
    try {
        const result = await Promise.reject('plain message')
    } catch (e) {
        console.log(`error message: ${e.message}`)
        console.log(`error: ${e}`)
    }
}
test()
```

Reply:
oh dang, you right. -didn't consider this

cool - prob can standardize/clean this up a bit but all good